server/crashmanager/management/commands/bug_update_status.py

```diff
@@ -58,3 +59,18 @@ def handle(self, *args, **options):
                     elif bug.closed is None:
                         bug.closed = bugStatus[bugId]
                         bug.save()
+
+            for bugId in bugIds:
+                if bugId not in bugStatus:
+                    bugs = providerBugs.filter(externalId=bugId)
+                    for bug in bugs:
+                        notify.send(
+                            bug,
+                            recipient=bug.tools_filter_users,
+                            actor=bug,
+                            verb="inaccessible_bug",
+                            target=bug,
+                            level="info",
+                            description=f"The bug {bug.pk} pointing to the external bug {bug.externalId}"
```

Comment:
```suggestion
                            description=f"Bucket {bug.pk} associated with bug {bug.externalId}"
```

Reply:
That's quite confusing to have the Bug internal pk after "Bucket" (`Bucket {bug.pk}`) no?

Reply:
I opened #727 to followup on that, this does not seem blocking as it's "only" wording and can be easily updated at a later stage.